tools/discovery/api.yaml.erb

```diff
@@ -0,0 +1,37 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO(nelsonjr): Make all Zone and Region resource ref
+
+--- !ruby/object:Api::Product
+name: Google Cloud <%= ctx[:product].title %>
+prefix: g<%= ctx[:product].title.split[0].downcase %>
+versions:
+  - !ruby/object:Api::Product::Version
+    name: <%= ctx[:product].version %>
+    base_url: <%= ctx[:product].base_url %>
```

Comment:
Note that, at least for compute, this includes "/projects" and the per-resource ones don't. This is probably ok?

Reply:
I think so? I can look into this more closely.